package.json

```diff
@@ -125,7 +125,11 @@
     "husky": "^4.2.5",
     "jscodeshift": "^0.7.0",
     "lint-staged": "^10.2.2",
-    "storybook-chromatic": "^3.4.1"
+    "storybook-chromatic": "^3.4.1",
+    "glob": "^7.1.1",
+    "loader-utils": "^1.0.2",
+    "node-elm-compiler": "^5.0.0",
+    "yargs": "^6.5.0"
```

Comment:
These are required for the forked version of `elm-wepack-loader`. Originally I tried to make a force a yarn workspace at `packages/elm-webpack-loader` (with `private: true` so it wouldn't get published), and this worked in local developement, but the CI publish script couldn't find it. I gave up and put it in `storybook/elm-webpack-loader-fix.js`